

Allow INIT_TASK_PAGG to be followed by something else in the INIT_TASK macro.


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/pagg.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN include/linux/pagg.h~process-aggregates-macro-fix include/linux/pagg.h
--- 25/include/linux/pagg.h~process-aggregates-macro-fix	2004-08-06 00:38:08.842534448 -0700
+++ 25-akpm/include/linux/pagg.h	2004-08-06 00:38:13.994751192 -0700
@@ -183,7 +183,7 @@ static inline void pagg_exec(struct task
  */
 #define INIT_TASK_PAGG(tsk) \
 	.pagg_list = LIST_HEAD_INIT(tsk.pagg_list),     \
-	.pagg_sem  = __RWSEM_INITIALIZER(tsk.pagg_sem)
+	.pagg_sem  = __RWSEM_INITIALIZER(tsk.pagg_sem),
 
 #else  /* CONFIG_PAGG */
 
_
